gdb-xml/tricore-core.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
```

Comment:
Please add a comment here where you got it from or if you created it and with what reference. Also for the other xml files.